ktor-shared/ktor-serialization/ktor-serialization-kotlinx/ktor-serialization-kotlinx-xml/gradle.properties

```diff
@@ -5,3 +5,5 @@
 # xmlutil uses DOM API under the hood so it doesn't support Node.js.
 # Issue: https://github.com/pdvrieze/xmlutil/issues/83
 target.js.nodeJs=false
+# xmlutil doesn't provide android native target
```

Comment:
Let's create an issue for the `xmlutil` and link here
I don't think it should be hard to support there

Reply:
Created: https://github.com/pdvrieze/xmlutil/issues/240